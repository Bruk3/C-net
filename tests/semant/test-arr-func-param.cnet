int print_arr(int[] a) {}

int main() {
    int[] x = new int[10]; 
    print_arr(x);
}