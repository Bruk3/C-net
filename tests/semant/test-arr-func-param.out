int print_arr(int[] a)
{
}

int main()
{
int[] x = (int[] : new int[(int : 10)] = {});
(int : print_arr((int[] : x)));
}

